app/components/sort-header/template.hbs

```diff
@@ -1,6 +1,9 @@
-{{yield}}
-{{#if this.active}}
-  <i class="fa fa-sort-{{this.direction}}"></i>
-{{else}}
-  <i class="fa fa-sort"></i>
-{{/if}}
+{{! template-lint-disable}}
```

Comment:
What would template-lint criticize?

Reply:
> app/components/sort-header/template.hbs
  1:24  warning  Interaction added to non-interactive element  no-invalid-interactive